encryption_works.md

```diff
@@ -98,25 +98,41 @@ Tails, which is a live DVD and live USB GNU/Linux distribution that I will discu
+Recently, the Tor Project [commented](https://blog.torproject.org/blog/preliminary-analysis-hacking-teams-slides) on reports that Hacking Team, the surveillance technology vendor which was massively hacked in July 2015, had managed to compromise the Tor network. The attack turned out to be a case of targeted surveillance rather than a widespread problem:
 
-The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en).
+> The good news is that they don't appear to have any exploit on Tor or on Tor Browser. The other good news is that their proposed attack doesn't scale well. They need to put malicious hardware on the local network of their target user, which requires choosing their target, locating her, and then arranging for the hardware to arrive in the right place. So it's not really practical to launch the attack on many Tor users at once.
 
-![](https://raw.github.com/micahflee/encryption-works/master/images/torbrowser.png)
+As with all privacy-enhancing software, an attacker with physical access to the device can compromise you in any number of ways.
 
-When Snowden [was answering questions on Guardian's website](http://www.guardian.co.uk/world/2013/jun/17/edward-snowden-nsa-files-whistleblower) from a "secure Internet connection", he was probably routing his traffic through the Tor network. He may have also been using a [bridge](https://bridges.torproject.org/) to connect to the Tor network to make the fact that he was using Tor from his IP address less obvious to eavesdroppers.
+### Setup
+
+The easiest way to start using Tor is to download and install the [Tor Browser Bundle](https://www.torproject.org/download/download-easy.html.en). 
+
+### Browsing Habits
+
+You should use Tor differently than you use a regular browser: some of the features it uses to obscure your location can be undermined if you don't take a few extra steps to protect yourself.
+
+- Don't enable or install browser plugins, including Adobe Flash. These can bypass Tor's security and give away your location.
+- Be aware of unencrypted sites, which don't have a lock icon next to the URL. Tor anonymizes your Internet traffic but unencrypted connections can still be eavesdropped on between the final node and the Internet server.
+- Don't download documents downloaded through Tor, since these documents can reveal your regular IP address to an adversary. Open them while offline or consider using Tails.
```

Comment:
This is referencing PDF and MS Word exploits and executables. i.e. you're using Tor Browser on your Windows computer and open a file you downloaded in the default program be it an unpatched version of Adobe Reader or Microsoft Word, boom you're exposed.